scripts/list_boards.py

```diff
@@ -150,10 +150,10 @@ def find_arch2board_set_in(root, arches, board_dir):
     boards = root / 'boards'
 
     for arch in arches:
-        if not (boards / "boards_legacy" / arch).is_dir():
+        if not (boards / arch).is_dir():
             continue
 
-        for maybe_board in (boards / "boards_legacy" / arch).iterdir():
+        for maybe_board in (boards / arch).iterdir():
```

Comment:
With this change by itself, all Arm boards will appear as both v1 and v2:
```
> west boards | grep mps2
mps2_an385
mps2_an521_cpu0
mps2_an521_cpu0_ns
mps2_an521_cpu1
mps2
```

Reply:
Fixed.